### ISSUE TITLE
Page Title Purpose - clarification

### ISSUE THREAD

**Issue by dderouchey** [Section508Coordinators/DHSA-TT-200-A#326]
[If you haven't already, please review the [Guidelines for Contributing](contributing.md) in the main folder of this repository before submitting any issues. You can delete this paragraph and other text that is not necessary before submitting your issue.]

## Issue Type (put a capital X between the brackets to indicate issue type)
- [ ] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [X] Other, specify...[additional scenario]

## Description
[If there is a set of web pages that use the same page title along with page numbering, will that pass the 12B test?]

## Location
[https://training.section508testing.net/mod/book/view.php?id=1254&chapterid=2977, Topic 12, Lesson 2: Page Title Purpose, Evaluating content]

## Actual Behavior / Observation
[An example might be a book or online course:
**Title:** Page 1 of 3; Social Media, Rules of Behavior
**Title;** Page 2 of 3; Social Media, Rules of Behavior
**Title;** Page 3 of 3; Social Media, Rules of Behavior]

**Comment by drewnielson** [comment]
Yes, the text along with the page numbering (page X of X) would be considered sufficient to differentiate it from other pages. We'll look for a good place to include that clarification in the course.
